apis/keda/v1alpha1/scaledobject_types.go

```diff
@@ -284,7 +284,7 @@ func CheckFallbackValid(scaledObject *ScaledObject) error {
 
 	for _, trigger := range scaledObject.Spec.Triggers {
 		if trigger.Type == cpuString || trigger.Type == memoryString {
-			return fmt.Errorf("type is %s , but fallback it is not supported by the CPU & memory scalers", trigger.Type)
+			return fmt.Errorf("type is %s, but fallback it is not supported by the CPU & memory scalers", trigger.Type)
 		}
 		if trigger.MetricType != autoscalingv2.AverageValueMetricType {
```

Comment:
Thanks for the reminder ping on this PR.

I think the issue being addressed in your PR is different from this one, so it would be better to have separate PRs to keep the changelog clean. From what I understand, my fix is for cases where the admission webhook is not rejecting objects which are invalid, while the issue you discovered is when the fallback is being incorrectly disabled due to the lack of `metricType`.

Does that sound okay with you?